apps/meteor/app/livechat/imports/server/rest/appearance.ts

```diff
@@ -62,3 +90,20 @@ API.v1.addRoute(
 		},
 	},
 );
+
+function coerceInt(value: string | number | boolean): number {
+	if (typeof value === 'number') {
+		return value;
+	}
+
+	if (typeof value === 'boolean') {
+		return 0;
```

Comment:
Are we okay with assuming 0 when an invalid value is provided as a number? If so, can we add some test cases to cover this and the `NaN` cases?